runtime/observability/telemetry-api/common/src/aws/smithy/kotlin/runtime/telemetry/trace/SpanContext.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.telemetry.trace
+
+/**
+ * The immutable state that must be serialized and propagated as part of a distributed trace context.
+ */
+public interface SpanContext {
+    public companion object {
+        public val Invalid: SpanContext = InvalidSpanContext
```

Comment:
When would an "invalid" SpanContext ever be used/desired? If it's meant to be used as a default when no context is set, I'd avoid the word "invalid"